assets/translations/en.json

```diff
@@ -125,6 +125,17 @@
     "wallet_header_title": "Wallet",
     "create_account_button_label": "Random account",
     "import_account_button_label": "Import account",
+    "contacts_button_label": "Contacts",
```

Comment:
Please always add new lines to the end of the file. I hope this way we will make localization support easier for translators